core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -132,13 +132,21 @@
     Optional<PlayerVideoTrack> getSelectedVideoTrack() throws IllegalStateException;
 
     /**
-     * Retrieves all of the available {@link PlayerVideoTrack} of a prepared Player.
+     * Clears the {@link PlayerVideoTrack} selection made in {@link NoPlayer#selectVideoTrack(PlayerVideoTrack)}.
      *
-     * @return a list of available {@link PlayerVideoTrack} of a prepared Player.
+     * @return whether the clear was successful.
+     * @throws IllegalStateException - if called before {@link NoPlayer#loadVideo(Uri, ContentType)}.
+     */
+    boolean clearVideoTrackSelection() throws IllegalStateException;
```

Comment:
is there thought to whether we'll remove thrown exceptions on APIs where we return boolean to indicate success?

Reply:
Good point. The `IllegalStateException` is used when a client calls any of these methods before the `player` is prepared, I guess we could do away with the throws in these cases but then the reasoning as to why the method returned false would be hidden. I guess you could argue the reasoning would be hidden anyway, so why return a `boolean` at all 😄  

I think you're right, this needs more thought 👅

Reply:
we need a trilean `true, false, error` 💃